### ISSUE TITLE
ChatGPT is only displaying partial results on DuckDuckGo results page

### ISSUE THREAD

**Issue by dnpage** [adamlui/userscripts#9]
![image](https://user-images.githubusercontent.com/2123779/218878108-1cc68a3f-3052-4dc7-985b-90d51516ef97.png)
Only a portion of the ChatGPT results are being displayed. This is with Chrome Version 109.0.5414.121 (Official Build) (64-bit)

**Comment by adamlui** [comment]
Can you goto chat.openai.com, do you see the query in your history? If so, can you click and see if ChatGPT typed the full answer?

Also, can you try running with all other userscripts disabled? If it works, can you re-enable them one by one and tell me the name of the script when it stops working?

Here's the full output of a sample reply in case you needed it:

![image](https://user-images.githubusercontent.com/10906554/218948094-f96e86e5-808f-424c-ae0b-8f4b05910c55.png)

**Comment by dnpage** [comment]
Thank  you for your quick response. I installed DuckDuckZgpT on Firefox 
and everything worked as expected. I went back to Chrome, Chrome did an 
auto-update and I tried DuckDuckGo and it started working. I had deleted 
all my previous chat history so I was not able to go back to see what 
they were generating. Either way, it is working for me on both browsers 
now. Thanks for providing making it possible for us to be early adaptors!
